neurom/core/_neuron.py

```diff
@@ -113,8 +114,15 @@ def iter_segments(obj, neurite_filter=None):
                                for sec in sections)
 
 
+def graft_neuron(root_section):
```

Comment:
I wonder whether there should be a recenter option, so that the soma is located at the origin?

Reply:
You mean, take the soma from the original neuron and translate it at the new root ?

Reply:
We could add a optional soma argument so the user can provide his own soma if he wants to.